runtime.md

```diff
@@ -103,6 +103,7 @@ This operation MUST generate an error if it is not provided the container ID.
 Attempting to start a container that does not exist MUST generate an error.
 Attempting to start an already started container MUST have no effect on the container and MUST generate an error.
 This operation MUST run the user-specified code as specified by [`process`](config.md#process-configuration).
+If `process.args` was not configured, the runtime MUST generate an error.
```

Comment:
I'd rather not make that change.  The execution will usually be by a call to `execve(2)` or similar, and while that returns errors like `ENOENT`, `ENOEXEC`, etc. on failure, there's no way to get it to return something on success (because by that point the process will be running the user-specified program).  And `start` is just signalling an external process to execute code (e.g. see opencontainers/runC#886, where the `start` process drains a fifo to trigger an `execve` in the container process).  So I don't see a way to block on potential `execve` errors in the `start` process short of `ptrace(2)` monitoring, and that seems like too high a cost.

I'd rather have `start` error out if it failed to *trigger* the start attempt (which the container process can confirm with “start received.  I'm about to `execve`, wish me luck”).  Folks who are curious about whether that `execve` succeeded can either `ptrace` the container process themselves,  or monitor `state`, etc.

[1]: https://github.com/opencontainers/runc/blob/v1.0.0-rc2/start.go#L30